### ISSUE TITLE
Making a specific gridster-item pushable but not draggable

### ISSUE THREAD

**Issue by sonulchoi217** [tiberiuzuld/angular-gridster2#375]
Thank you for the library! It works really well, and I love that a lot of options are customizable.

I'm trying to make one of the gridster-item pushable, but not dragable. It seems like if you make an item not draggable, it also becomes not pushable. Is there a workaround for this issue?

Thank you so much!

**Comment by Achilles1515** [comment]
@sonulchoi217 I don't believe there is a workaround because the push() function explicitly checks to make sure the item is draggable, so you'd have to edit the code.

https://github.com/tiberiuzuld/angular-gridster2/blob/master/projects/angular-gridster2/src/lib/gridsterPush.service.ts#L130

@tiberiuzuld Is there a specific reason why only draggable items can be pushed? The use case I imagined was having nothing draggable, but items can still be pushed around by resizing. I removed that `if` statement linked above entirely and this functionality is enabled.

**Comment by tiberiuzuld** [comment]
Hi @Achilles1515 ,
Yes only the items which are draggable can be pushed.
This is because in most cases or most common sense is that if you don't want an item draggable/moved then you don't want it pushed.
